composer.json

```diff
@@ -44,12 +44,13 @@
     "dealerdirect/phpcodesniffer-composer-installer": "^0.7.0",
     "php-stubs/wp-cli-stubs": "^2.4",
     "phpunit/phpunit": "^7.5.20",
```

Comment:
Maybe worth using `yoast/wp-test-utils` here or changing the version constraint to ensure packages can be installed on PHP 8.x

Reply:
Created #237 for this